docs/org-units/getting-web-api.ipynb

```diff
@@ -0,0 +1,257 @@
+    "\n",
+    "### List OUs by level (auto‑paged)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "72942972",
+   "metadata": {
+    "scrolled": true,
+    "tags": [
+     "scroll-output",
+     "output_scroll"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "▶ Fetching orgunits at level 3 (first 10 ous shown as sample):\n",
+      "YuQRtpLP10I · Badjia\n",
+      "jPidqyo7cpF · Bagruwa\n",
+      "vWbkYPRmKyS · Baoma\n",
+      "dGheVylzol6 · Bargbe\n",
+      "zFDYIgyGmXG · Bargbo\n",
+      "RzKeCma9qb1 · Barri\n",
+      "EB1zRKdYjdY · Bendu Cha\n",
+      "fwH9ipvXde9 · Biriwa\n",
+      "ENHOJz3UH5L · BMC\n",
+      "KKkLOTpMXGV · Bombali Sebora\n",
+      "… (more not shown)\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "print(\"▶ Fetching orgunits at level 3 (first 10 ous shown as sample):\")\n",
+    "for ou in islice(client.get_organisation_units(level=3, fields=\"id,name\", order=\"name:asc\"),10):\n",
+    "    print(f\"{ou['id']} · {ou['name']}\")\n",
+    "print(\"… (more not shown)\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f86a7c78",
+   "metadata": {},
+   "source": [
+    "### Fetch all OUs in the system"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "f349b553",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "▶ Total number of ous available:  1332\n"
+     ]
+    }
+   ],
+   "source": [
+    "ous = list(client.get_organisation_units())\n",
+    "print(\"▶ Total number of ous available: \", len(ous))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35ec29d2",
+   "metadata": {},
+   "source": [
+    "`client.get_organisation_units()` returns an `Iterable` and can be converted to a `List` of Dict for further processing. However, please be careful of memory usage if you have a huge number of org units in your system.\n",
+    "\n",
+    "While `.get_organisation_units(...)`is a convenient method to faciliate page by page data fetching, the library also allows a raw access where you can pass filters and params direct to DHIS2 API. Below are some examples:\n",
+    "\n",
+    "### Get a single OU by UID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "269379bf",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "▶ Fetched org unit O6uvpzGd5pu: {'name': 'Bo', 'parent': {'id': 'ImspTQPwCqd'}, 'id': 'O6uvpzGd5pu', 'level': 2}\n"
+     ]
+    }
+   ],
+   "source": [
+    "ou_id = \"O6uvpzGd5pu\"  # Change this to your desired org unit ID\n",
+    "fields = \"id,name,level,parent\"\n",
+    "org_unit = client.get(\"/api/organisationUnits/\" + ou_id, params={\"fields\": \"id,name,level,parent\"})\n",
```

Comment:
Not crucial, but since you're already using f-strings, maybe use that here too: `f"/api/organisationUnits/{ou_id}"`.